### ISSUE TITLE
New pick screen doesn't work

### ISSUE THREAD

**Issue by chrisinajar** [OpenAngelArena/oaa#1408]
Currently the pick screen works when you use it locally, I even taught the bots how to use CM so that we can test that UI. It's lovely and not entirely complete but works* and it's ours.

However, when you upload the code into a game mode everything freaks out.

You sit at that "something broke" screen we all know; where the screen is over the center of the map and you don't have a hero and you can scroll around wherever you want but everything is covered in fog... You know, that one. What's actually happening is the default dota pick screen is running, and if you disconnect then reconnect you'll see the UI and can pick a hero. Once this screen has timed out (which also almost certainly burns gold, but i didn't test that) it does the tower thing like normal**.

After the tower thing it finally drops you into the new fancy custom pick screen. This pick screen, for whatever reason, isn't populated and doesn't do anything.

Basically it doesn't work online the way it works locally at all.

This is a bug that requires repeatedly uploading the game mode and trying silly shit since it works fine locally already and that's the only way to test it.

This is a hard blocker preventing new versions*** of the game from being released.

\* It doesn't
** This also doesn't happen locally so dealing with coding around it is annoying
*** For the love of god nerf clinkz

**Comment by chrisinajar** [comment]
HAHA! FIXED MOFUKKKAAAAHHHHHH
